doc/admin-guide/logging/formatting.en.rst

```diff
@@ -605,6 +606,7 @@ pssc  Proxy Response        HTTP response status code sent by |TS| proxy to the
                             client.
 sssc  Origin Response       HTTP response status code sent by the origin server
                             to the |TS| proxy.
+prrr  Proxy Server          Proxy response result reason
```

Comment:
Is this the Message that goes along with the Status Code? eg, the "OK" in 200 OK, and Not Modified in "304 Not Modified"?  If so, wonder if "message phrase" or "response phrase" would make it more clear? (both seem to be used in rfc 7231.

Reply:
Good point. I've renamed it to "reason phrase" and updated things accordingly.